tools/mpremote/mpremote/commands.py

```diff
@@ -187,6 +392,11 @@ def _do_execbuffer(state, buf, follow):
     state.ensure_raw_repl()
     state.did_action()
 
+    def stdout_write_bytes(b):
```

Comment:
This assumes that stdout has a buffer, which is not the case if called from Jupyter 
proposed :
```
        if hasattr(sys.stdout, 'buffer'):
            sys.stdout.buffer.write(b)
            sys.stdout.buffer.flush()
        else:
            text = b.decode(sys.stdout.encoding, 'strict')
            sys.stdout.write(text)
 ```

Reply:
Done, thanks for the suggestion!